### ISSUE TITLE
Incompatibility with newer versions of kotlin wrappers

### ISSUE THREAD

**Issue by AKJAW** [cfnz/muirwik#29]
I tried adding Muirwik the way it is done in the [starter](https://github.com/cfnz/muirwik-starterapp). I got my app to compile but I always had the same error **TypeError: $receiver.child_2usv9w$ is not a function**

After a little bit of digging around I found out that this was caused by the createStyled function inside **muirwik-muirwik-components.js**. The **child_2usv9w$** refers to a prototype field inside kotlin-react which changes depending on the wrapper version. 
kotlin-react:16.13.1-pre.104-kotlin-1.3.72 - has the name **child_2usv9w**
kotlin-react:16.13.1-pre.105-kotlin-1.3.72 - has the name **child_52psg1$**
This name discrepancy causes Muirwik to fail at runtime. 

I think it is worth mentioning this compatibility issue in the README, along with the needed dependency setup needed to get Muirwik working and maybe a link to the starter project. If you want I can create a pull request with these suggestions

**Comment by cfnz** [comment]
Yip, version incompatibility has caused me some grief in the past... 
Fire through a pull request and I shall take a look... any help appreciated :-).

**Comment by AKJAW** [comment]
Sorry for the delay, I have created a pull request with the updated README 
https://github.com/cfnz/muirwik/pull/30

**Comment by NikkyAI** [comment]
i hope you can make eap variants of the library so people can make use of the better dce in `1.4-M2`

i fixed this locally for myself so i can use `1.4-M2`
https://github.com/NikkyAI/muirwik/blob/1.4-M2/muirwik-components/build.gradle.kts

i removed bintray and maven-publications because i do not use them myself and the parent prject cannot provide them

this is how it is included:

added as git-submodule

`.gitmodules`
```gitmodules
[submodule "muirwik"]
	path = muirwik
	url = https://github.com/NikkyAI/muirwik.git
```

`settings.gradle.kts`
```kt
// include muirwik
include(":muirwik")
project(":muirwik").projectDir = rootDir.resolve("muirwik/muirwik-components")
```

dependency: `implementation(project(":muirwik"))`
